scripts/migration/statefulset/generate-certs.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+mkdir -p certs
+mkdir -p backup
+
+# Fetch and remap CA cert.
+kubectl get secret -o yaml $RELEASE_NAME-cockroachdb-ca-secret | yq '.data."ca.key"' | base64 -d >certs/ca.key
+kubectl get secret -o yaml $RELEASE_NAME-cockroachdb-node-secret | yq '.data."ca.crt"' | base64 -d >certs/ca.crt
+kubectl create configmap $RELEASE_NAME-cockroachdb-node-ca --from-file=certs/ca.crt
+kubectl create configmap $RELEASE_NAME-cockroachdb-client-ca --from-file=certs/ca.crt
```

Comment:
The assumption that the generation of the certificate secret names based on "release_name-cockroachdb" is not entirely correct.

If your release name consist of cockroachdb then -cockroachdb suffix is not added. Logic [here](https://github.com/cockroachdb/helm-charts/blob/116b55954abd627800b2f319d1faca68134a27dc/cockroachdb/templates/_helpers.tpl#L17)

You need to add following code to create a name:
```
base_chart_name="cockroachdb"
if [[ "$release_name" == *"$base_chart_name"* ]]; then
    export crdb_name="${release_name}"
  else
    export crdb_name="${release_name}-${base_chart_name}"
fi
``` 

Now replace crdb_name with RELEASE_NAME-cockroachdb